paasta_tools/paasta_metastatus.py

```diff
@@ -271,12 +271,17 @@ def fill_table_rows_with_service_instance_stats(
     limiting_factor = "Unknown"
     # service_instance_stats.keys() should be a subset of resource_free_dict
     for rsrc_name, rsrc_amt_wanted in service_instance_stats.items():
-        if rsrc_amt_wanted > 0:
+        if rsrc_amt_wanted > 0:  # type: ignore
```

Comment:
rsrc_amt_wanted could be either int or float, and i know how to "fix" this. This behavior itself is not possible if we use static types.